templates/pages/service-levels/levelagreement.html.twig

```diff
@@ -0,0 +1,118 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2025 Teclib' and contributors.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+{% extends "generic_show_form.html.twig" %}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+ {% set params  = params ?? [] %}
+{# {% set edit_mode = edit_mode|default(false) %}#}
+
+{% block more_fields %}
+
+    {# slm - hidden field + link #}
+    {{ fields.hiddenField('slms_id', item.fields['slms_id']) }}
+    {% set slm_link %}
+        {% set slm = get_item('SLM', item.fields['slms_id']) %}
+        <a href="{{ slm.getFormURLWithID(item.fields['slms_id']) }}" class="text-decoration-none">
+            {{ slm.fields['name'] }}
+        </a>
+    {% endset %}
+    {{ fields.htmlField('no_matter', slm_link, __('SLM')) }}
+
+    {# lastupdate #}
+    {{ fields.datetimeField('last_update', item.fields['date_mod'], __('Last update'), {'disabled': true}) }}
+
+    {# type #}
+    {{ fields.dropdownArrayField(
+        'type',
+        item.fields['type'],
+        item.getTypes(),
+        _n('Type', 'Types', 1)
+    ) }}
+
+    {# maximumtime #}
+    {{ fields.dropdownArrayField(
+        'number_time',
+        item.fields['number_time'],
+        range(0, 1000),
+        ''
+    ) }}
+
+    {# @todo doesn't work, apply on group dropdown autogenerated, id is ok, but hidden #}
+    {% do call('Html::showToolTip', [__('Actions done by users of this group will trigger the TTO/TTR.'), {
```

Comment:
You could still use the `generic_show_form` for the overall layout but override the fields block to re-implement the Groups dropdown. Then, you can use the `helper` parameter to add this tooltip.

Reply:
fixed, thanks.